media-gfx/flameshot/metadata.xml

```diff
@@ -9,4 +9,8 @@
 		<email>proxy-maint@gentoo.org</email>
 		<name>Proxy Maintainers</name>
 	</maintainer>
+	<use>
+		<flag name='system-singleapplication'>Use the system-wide
+			<pkg>dev-qt/qtsingleapplication</pkg> instead of bundled.</flag>
+	</use>
```

Comment:
Since you're unbundling the dep already with a patch, why would we want to use the bundled one? I think this doesn't need to be a choice, just always prefer the system one.

Reply:
See the patch. It adds ifdefs and cmake flag whether to use system-provided (I think I will try to add this to mainstream).
The bundled singleapplication version is actually different from the system one, they should be compatible, but who knows. I will probably make this USEFLAG default selected.

Reply:
The patch is fine, you could just always use `-DUSE_EXTERNAL_SINGLEAPPLICATION=1` in `src_configure` while removing the bundled one nevertheless. I'd say let's always prefer the system one at least until it breaks, and even then, the choice is visible in the ebuild to quickly test whether bundled one still works.